src/components/BasicInfo.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledFieldset from './StyledFieldset';
+
+export default function BasicInfo({
+  firstName,
+  lastName,
+  pronouns,
+  setRegisterStep,
+  handleFormChange,
+}) {
+  const filledOut = firstName !== '' && lastName !== '' && pronouns !== '' && true;
+  return (
+    <StyledFieldset>
+      <legend>Basic Info</legend>
+      <p>Input basic info about yourself.</p>
+      <p className="warning">*Fields are required</p>
+      <label htmlFor="firstName">
+        First Name<span className="warning">*</span>
```

Comment:
Bonus points if you do this: https://a11y-101.com/development/required

Reply:
You know I'm always going to try for bonus points, but it seems the topic is more complex than I would have thought. Let me know what you think of what I settled on. Otherwise do you think it'd make more sense to fall back to an image solution? Or maybe it's worth asking Joseph (like the article suggests) if he'd want to change it to simply say "required"?

Reply:
No worries--I'll turn this into a separate ticket